{"name": "test_add2[【类型】输入特殊字符，给出提示信息1]", "status": "passed", "steps": [{"name": "步骤一：*&和0.2开始计算，预期应等于参数大小超出范围，打印<ExceptionInfo for raises contextmanager>", "status": "broken", "statusDetails": {"message": "TypeError: '>' not supported between instances of 'str' and 'int'\n", "trace": "  File \"D:\\zj\\PycharmProjects\\study202208\\study0818\\test\\test_calculate.py\", line 52, in test_add2\n    result=self.cal.add(x,y)\n  File \"D:\\zj\\PycharmProjects\\study202208\\script\\calculator.py\", line 4, in add\n    if a > 99 or a < -99 or b > 99 or b < -99:\n"}, "start": 1660804986841, "stop": 1660804986841}], "attachments": [{"name": "log", "source": "3aea6416-9548-4df5-a895-8785a745a69f-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "30b0980b-d55c-4152-8d14-7319c0697d51-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "d995411c-7ecf-4511-854d-b30866a46630-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "x", "value": "'*&'"}, {"name": "y", "value": "0.2"}, {"name": "expect", "value": "'参数大小超出范围'"}], "start": 1660804986840, "stop": 1660804986841, "uuid": "e64c1fd1-a7ee-4a0e-8a94-ff7c505feec7", "historyId": "14c868acb194d8087bf412026f7ce2df", "testCaseId": "c1fd4152c4894ab47d6f7448dc1c5bcf", "fullName": "study0818.test.test_calculate.TestAdd#test_add2", "labels": [{"name": "story", "value": "加法报错用例"}, {"name": "tag", "value": "P2"}, {"name": "parentSuite", "value": "study0818.test"}, {"name": "suite", "value": "test_calculate"}, {"name": "subSuite", "value": "TestAdd"}, {"name": "host", "value": "LAPTOP-BEO6NH7O"}, {"name": "thread", "value": "13372-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "study0818.test.test_calculate"}]}